notNeededPackages.json

```diff
@@ -804,6 +804,12 @@
             "sourceRepoURL": "https://github.com/localForage/localForage",
             "asOfVersion": "0.0.34"
         },
+        {
+            "libraryName": "localforage-cordovasqlitedriver module",
```

Comment:
Do we need the **module** suffix in the `libraryName`.
Didn't make sense to me & it also seems that no other `libraryName` contains that.